docs/content/guides/developer/cryptography/multisig.mdx

```diff
@@ -150,4 +152,5 @@ sui client execute-signed-tx --tx-bytes <TX_BYTES> --signatures <SERIALIZED-MULT
 
 ## Related links
 
-- [Multisig](../../../concepts/cryptography/transaction-auth/multisig.mdx): Definition of multisig transactions on the Sui network.
\ No newline at end of file
+- [Multisig](../../../concepts/cryptography/transaction-auth/multisig.mdx): Definition of multisig transactions on the Sui network and how to use it with CLI.
+- [Multisig Typescript SDK](https://sdk.mystenlabs.com/typescript/cryptography/multisig): How to create multisig address and sign transactions using the Sui SDK.
```

Comment:
not sure how to link the sdk page's original page, feel free to fix @ronny-mysten

Reply:
My guess is you've never not figured something out correctly. This case included ;)